dashboard/app/views/home/_survey_nps.html.haml

```diff
@@ -11,11 +11,11 @@
   }
 
 
-.survey{style: "background-color: rgb(200,200,200); padding:40px; font-size:18px; border-radius: 4px"}
+.survey{style: "background-color: rgb(200,200,200); padding:25px; font-size:14px; border-radius: 4px; margin-bottom: 60px"}
```

Comment:
super nit: we usually use a slightly lighter gray for backgrounds in teacher dashboard -- `rgb(236, 236, 236)` -- should we use the same one here?

example:
![Screen Shot 2019-03-25 at 9 44 23 AM](https://user-images.githubusercontent.com/9812299/54937909-9a4fb080-4ee2-11e9-88df-c9c6aa693c31.png)

Reply:
Great eye.  Tempting as it was to have some throwback UI, I modernised the styling.